tests/dialects/test_smt.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from xdsl.dialects.builtin import IntegerAttr
+from xdsl.dialects.smt import (
+    AndOp,
+    BoolType,
+    ConstantBoolOp,
+    OrOp,
+    VariadicBoolOp,
+    XOrOp,
+)
+from xdsl.utils.test_value import create_ssa_value
+
+
+def test_constant_bool():
+    op = ConstantBoolOp(True)
+    assert op.value is True
+    assert op.value_attr == IntegerAttr(-1, 1)
+
+    op = ConstantBoolOp(False)
+    assert op.value is False
+    assert op.value_attr == IntegerAttr(0, 1)
+
+
+@pytest.mark.parametrize("op_type", [AndOp, OrOp, XOrOp])
+def test_variadic_bool_op(op_type: type[VariadicBoolOp]):
+    arg1 = create_ssa_value(BoolType())
+    arg2 = create_ssa_value(BoolType())
+    arg3 = create_ssa_value(BoolType())
+    op = op_type([arg1, arg2, arg3])
```

Comment:
```suggestion
    op = op_type(arg1, arg2, arg3)
```

This should work right?

Reply:
I mean, what would you prefer here?
To build operations using `or(arg1, arg2, arg3)`, or `or([arg1, arg2, arg3])`?
I would agree that the first one seems nicer